guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,60 @@
+[id="Exporting_a_Repository_in_a_Syncable_Format_{context}"]
+= Exporting a Repository in a Syncable Format
+
+You can export the content of a repository in the Library environment of an organization to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content using a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports.
+Instead on the importing {ProjectServer} you must:
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optional: Set an SSL CA Credential if the web server requires it.
```

Comment:
In the other document, you write "SSL/TLS CA Credential". Maybe the duplicate block for Satellite should be moved to a snippet to avoid duplication.

The snippet may contain line 6 through 38.